dracut.sh

```diff
@@ -1295,6 +1296,13 @@ else
     exit 1
 fi
 
+if [[ $persistent_policy == "mapper" ]]; then
+    unset persistent_policy
```

Comment:
Can you help to understand this section ? Why do you unset persistent_policy ?

Reply:
See where the `persistent_policy` variable is used:

https://github.com/dracutdevs/dracut/blob/cf31fcf804be4dc0fa31885f5185a59b6012cdf4/dracut-functions.sh#L270-L290

By default (unset), the first check is `/dev/mapper/*`. With this patch I just want to give a name ("mapper") to the default option, because if any distro/package sets the `persistent_policy` in a conf file, and the user wants to change it to `/dev/mapper/*`, there is not a common way to do it (read PR description).

Reply:
Thanks !